cmd/juju/controller/listmodels_test.go

```diff
@@ -102,12 +106,21 @@ func (f *fakeModelMgrAPIClient) ListModelsWithInfo(user names.UserTag) ([]params
 			AgentVersion:       info.Result.AgentVersion,
 		}
 		if len(info.Result.Users) > 0 {
-			results[i].Result.User = info.Result.Users[0]
+			results[i].Result.UserAccess = info.Result.Users[0].Access
+			results[i].Result.UserLastConnection = info.Result.Users[0].LastConnection
 		}
-		// TODO (anastasiamac 2017-11-22) need to cater for cores count too
 		if len(info.Result.Machines) > 0 {
 			results[i].Result.Counts = []params.ModelEntityCount{
-				params.ModelEntityCount{params.Machines, len(info.Result.Machines)},
+				params.ModelEntityCount{params.Machines, int64(len(info.Result.Machines))},
```

Comment:
Same here.
Is this just because we don't want to have to change the Test data ?